src/shortcuts/cursorDown.tsx

```diff
@@ -53,11 +71,9 @@ const cursorDownShortcut: Shortcut = {
     const isProseMode = isProseView && window.getSelection()?.focusOffset as number < headValue(cursor).length - 1
     if (isProseMode) return false
 
-    // use default browser if selection is not on the last line of a multi-line editable
-    const isOnLastLine = isSelectionOnLastLine()
-    if (!isOnLastLine) return false
-
-    return true
+    const selectionAttributes = getSelectionAttributes()
+    // use default browser if there is a valid selection and it's not on the last line of a multi-line editable
+    return selectionAttributes ? isSelectionOnLastLine(selectionAttributes) : true
```

Comment:
This can be simplified to:

```js
return !selectionAttributes || isSelectionOnLastLine(selectionAttributes)
```

Boolean literals in conditionals are usually sign that the expression can be simplified. Though your use of the ternary is an improvement on what was there before!